test/common/tracing/zipkin/tracer_test.cc

```diff
@@ -26,31 +27,36 @@ class TestReporterImpl : public Reporter {
 TEST(ZipkinTracerTest, spanCreation) {
   Network::Address::InstanceConstSharedPtr addr =
       Network::Address::parseInternetAddressAndPort("127.0.0.1:9000");
-  Tracer tracer("my_service_name", addr);
-  MockMonotonicTimeSource mock_start_time;
-  MonotonicTime start_time = mock_start_time.currentTime();
+  Runtime::RandomGeneratorImpl random_generator;
```

Comment:
Yes, please remove commented out code. The point of the mock is so that you can return whatever value want, and can exercise all of your code paths completely and reproducibly. So make sure you have full coverage when using all mocks, and remove all commented out checks.